packages/logs/logs.0.6.3/opam

```diff
@@ -16,10 +16,12 @@ depends: [
 depopts: [
   "js_of_ocaml"
   "fmt"
-  "cmdliner" {>= "1.0.4."}
+  "cmdliner"
   "lwt" ]
 conflicts: [
-  "js_of_ocaml" { < "3.3.0" } ]
+  "js_of_ocaml" { < "3.3.0" } 
+  "cmdliner" { < "1.0.0" }
```

Comment:
```suggestion
  "cmdliner" { < "0.9.8" }
```
`Cmdliner.Arg.env` was added in 0.9.8: https://github.com/dbuenzli/cmdliner/commit/e82fd02842719167c3a04fc7ed0edf9a8036203c

Reply:
```
-> installed ocamlfind.1.9.1
-> installed ocamlbuild.0.14.0
-> installed cmdliner.0.9.8
-> installed topkg.1.0.4
-> installed logs.0.6.3
```
Tested on OCaml 4.05. It seems to work fine.